manifests/broker/config.pp

```diff
@@ -8,7 +8,6 @@
 # It manages the broker config files
 #
 class kafka::broker::config(
-  $install_dir = $kafka::broker::install_dir,
```

Comment:
This change does not justifiy a major version upgrade, because this parameter is not used anywhere.